.gitignore

```diff
@@ -93,8 +93,9 @@ $RECYCLE.BIN/
 # Source maps
 *.map
 
-# We don't need to keep the Swagger UI code under revision control
-docs/*
```

Comment:
I still maintain that we should only keep our project-specific code under revision control. Any auto-generated files or static dependencies should be tracked externally, when possible.